schedule/virt_autotest/slem_open_vm_tools.yaml

```diff
@@ -0,0 +1,30 @@
+name:           slem_open_vm_tools
+description:    >
+    Maintainer: nan.zhang@suse.com, qe-virt@suse.de
+    SLE-Micro vmdk image setup with integration services and open-vm-tools test modules
+schedule:
+    - '{{bootloader}}'
+    - jeos/firstrun
+    - transactional/host_config
+    - console/suseconnect_scc
+    - console/system_prepare
+    - console/check_network
+    - console/system_state
+    - console/integration_services
+    - '{{open_vm_tools}}'
+conditional_schedule:
+    bootloader:
+        VIRSH_VMM_FAMILY:
+            vmware:
+                - installation/bootloader_svirt
+                - '{{uefi}}'
+    uefi:
+        UEFI:
+            1:
+                - installation/bootloader_uefi
+            0:
```

Comment:
SLE Micro  only supports UEFI, so this legacy part can be dropped , I think :).

Reply:
Updated, thanks:)